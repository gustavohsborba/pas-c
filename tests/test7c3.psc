var
    s, p is int;
    pudim is int;

begin
    s := 0;

    do
        out({Valor a ser somado: });
        in(p);
        s := s+p;
    while p<>0;

    out({Total a pagar: R$});
    out(s);
end