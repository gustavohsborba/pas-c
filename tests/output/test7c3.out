Opening file tests/test7c3.psc...
compilation finished with success
